app/src/main/java/org/wikipedia/util/BatchUtil.kt

```diff
@@ -0,0 +1,48 @@
+package org.wikipedia.util
+
+import org.wikipedia.Constants
+import org.wikipedia.page.PageTitle
+import kotlin.math.min
+
+object BatchUtil {
```

Comment:
Looks like we can remove this file since this `BatchUtil` is only used on the test case itself.

Reply:
Yes @cooltey I saw that, but I thought may be we could use it as a central place for batch processing... but I see your point... let us introduce it if we decide that and also would need to be able to support beyond just PageTitles.